userprofile/admin.py

```diff
@@ -21,26 +21,57 @@
 """
 
 # Imports #####################################################################
-
 from django.contrib import admin
 from django.contrib.auth.admin import UserAdmin as BaseUserAdmin
 from django.contrib.auth.models import User
+from django.core.urlresolvers import reverse
+from django.utils import timezone
+from django.utils.html import format_html
 
 from userprofile.models import UserProfile, Organization
 
 
 # ModelAdmins #################################################################
 
-class UserProfileInline(admin.TabularInline): #pylint: disable=missing-docstring
+class UserProfileInline(admin.TabularInline):  # pylint: disable=missing-docstring
     model = UserProfile
     can_delete = False
     verbose_name_plural = 'profile'
 
 
-class UserAdmin(BaseUserAdmin): #pylint: disable=missing-docstring
+class UserAdmin(BaseUserAdmin):  # pylint: disable=missing-docstring
     inlines = (UserProfileInline,)
 
 
+class OrganizationAdmin(admin.ModelAdmin):  # pylint: disable=missing-docstring
+    list_display = (
+        'github_handle',
+        'account_actions',
+    )
+
+    def account_actions(self, obj):  # pylint: disable=no-self-use
+        """
+        This method will extract actions in the admin panel for each record. Currently
+        the only action we're generating is a link to the report.
+        :param obj: The instance we this action is related to
+        :return: An HTML of the actions we want to display.
+        """
+        now = timezone.now()
+
+        if obj:
+            return format_html(
+                '<a class="button" target="_blank" href="{}">Invoice Report</a>',
+                reverse('reports:report', kwargs={
+                    'organization': obj.github_handle,
```

Comment:
If an organization doesn't have a github handle, could you avoid the link? (otherwise it includes `/None/2018/10/`). You can add it to the if.